odoo/addons/base/models/ir_mail_server.py

```diff
@@ -556,6 +591,80 @@ def send_email(self, message, mail_server_id=None, smtp_server=None, smtp_port=N
             raise MailDeliveryException(_("Mail Delivery Failed"), msg)
         return message_id
 
+    @api.model
+    def _find_mail_server(self, mail_from):
+        """Find the appropriate mail server for the given email address.
+
+        Returns: Record<ir.mail_server>, mail_from
+        - Mail server to use to send the email (None if we use the odoo-bin arguments)
+        - Email FROM to use to send the email (in some case, it might be impossible
+          to use the given email address directly if no mail server is configured for)
+        """
+        normalized_mail_from = email_normalize(mail_from)
+        mail_from_domain = email_domain_extract(normalized_mail_from)
+        notifications_email = email_normalize(self._get_default_from_address())
```

Comment:
Is it default_from ? Isn't it a different parameter ? #abitlost

Reply:
Yes it's the default from, but I also use it to encapsulate the email when needed (so this parameter is used when no "FROM" is specify (like before my changes) or as a "notifications@domain.com" email).